docs/2024/puzzles/day03.md

```diff
@@ -22,48 +22,57 @@ https://adventofcode.com/2024/day/3
 
 This task plays very well with Scala's strengths: pattern matching and the standard collections library. But first, we need to learn a bit of regular expressions. Personally, I believe everyone should learn them. They're scary only at the beginning, and then they become a very valuable tool in your pocket.
 
-The shopkeeper at the North Pole Toboggan Rental Shop asks us to go through the input string, find all substrings that look like calls to a multiplication method `"mul(a,b)"`, perform the multiplications, and sum them together. To find such  substrings, we will construct a regular expression that will look for the following:
+The shopkeeper at the North Pole Toboggan Rental Shop asks us to go through the input string, find all substrings that look like calls to a multiplication method `"mul(a,b)"`, perform the multiplications, and sum them together. To find such substrings, we will construct a regular expression that will look for the following:
+
 1. Each substring starts with a text "mul(". We need to escape the parenthesis in the regex form, so this will be `mul\(`.
 2. "mul(" is followed by the first positive integer. Since it's an integer and positive, we don't need to look for floating points or minuses; we can look for sequences of ciphers. In regex, one cipher is denoted by `\d`, and since we look for a sequence of one or more ciphers, it's `\d+`.
 3. After the first number there should be a comma, so `,`.
 4. Then there should be a second number, so again `\d+`.
 5. And the substring should finish with the closing parentheses, escaped again, so `\)`.
-That's it. Our whole regex looks like this: 
+   That's it. Our whole regex looks like this:
```

Comment:
I think this was intended to be a new paragraph, not part of point 5?

Reply:
ah could be! fixing it now